src/rl/models/gnn_encoder.py

```
@@ -0,0 +1,137 @@
+
+        return x
+
+    def forward_batch(
+        self,
+        x: torch.Tensor,
+        edge_index: torch.Tensor,
+        edge_weight: torch.Tensor,
+        node_mask: torch.Tensor,
+        edge_mask: torch.Tensor,
+    ) -> torch.Tensor:
+        batch_size, num_nodes, feat_dim = x.size()
```

Ruff (0.14.2)

Unpacked variable `feat_dim` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)